Changelog.md

```diff
@@ -1,3 +1,18 @@
+# 0.8.0.0
+
+- GHC 8.6.4 support added.
+- Resolver bumped, LTS 13.10 for GHC 8.6.3, LTS 13.15 for GHC 8.6.4 (@alanz)
+- Clarify install section of README.md (@Anrock, @antonlogvinenko)
+- Clarify the spacemacs installation (@chkl)
+- Further install.hs improvements
+  - idempotent builds (@fendor)
+  - remove redundant shake.yaml (@fendor)
```

Comment:
`shake.yaml` is still present and needed.

Reply:
so it is, I think it was removed and then came back, I just went through the commits